File: JAVA/DPR.java
public class DPR
{
    // private atribut yang dibutuhkan
    private String id;
    private String name;
    private String bidang;
    private String partai;

    // constructor 
    public DPR()
    {
        this.id = "";
        this.name = "";
        this.bidang = "";
        this.partai = "";
    }
    public DPR(String id, String name, String bidang, String partai)
    {
        this.id = id;
        this.name = name;
        this.bidang = bidang;
        this.partai = partai;
    }

    // getter dan setter
    public String getId()
    {
        return this.id;
    }
    public void setId(String id)
    {
        this.id = id;
    }
    
    public String getName()
    {
        return this.name;
    }
    public void setName(String name)
    {
        this.name = name;
    }
    
    public String getPartai()
    {
        return this.partai;
    }
    public void setPartai(String partai)
    {
        this.partai = partai;
    }
    
    public String getBidang()
    {
        return this.bidang;
    }
    public void setBidang(String bidang)
    {
        this.bidang = bidang;
    }
}

File: PHP/DPR.php
<?php
    class DPR 
    {
        // atribut private yang dibutuhkan
        private $id = "";
        private $name = "";
        private $bidang = "";
        private $partai = "";
        private $foto = "";
        // constructor
        public function __construct($id = "", $name = "", $bidang = "", $partai = "", $foto = "")
        {
            $this->id = $id;
            $this->name = $name;
            $this->bidang = $bidang;
            $this->partai = $partai;
            $this->foto = $foto;
        }
        // getter dan setter untuk setiap atribut
        public function setId($id)
        {
            $this->id = $id; }
        public function getId()
        {
            return $this->id;
        }
        public function setName($name)
        {
            $this->name = $name;
        }
        public function getName()
        {
            return $this->name;
        }
        public function setBidang($bidang)
        {
            $this->bidang = $bidang;
        }
        public function getBidang()
        {
            return $this->bidang;
        }
        public function setPartai($partai)
        {
            $this->partai = $partai;
        }
        public function getPartai()
        {
            return $this->partai;
        }
        public function setFoto($foto)
        {
            $this->foto = $foto;
        }
        public function getFoto()
        {
            return $this->foto;
        }
    }
?>
